examples/ovirt_infra_vars.yml

```diff
@@ -0,0 +1,126 @@
+---
+###########################
+# REST API variables
+###########################
+engine_url: https://ovirt-engine.example.com/ovirt-engine/api
```

Comment:
should we change the url to include only FQDN? aren't the suffixes always the same (/ovirt-engine/api)?